tests/integration/query_graph/test_online_serving.py

```diff
@@ -16,7 +16,7 @@
 from tests.util.helper import create_batch_request_table_from_dataframe, fb_assert_frame_equal
 
 
-@pytest.fixture(name="features", scope="session")
+@pytest.fixture(name="features", scope="module")
```

Comment:
Update the scope here from `session` to `module` so that `mock_task_manager` (https://github.com/featurebyte/featurebyte/blob/898c4be052755ea3517c248da6ed5de78130d594/tests/integration/conftest.py#L1303-L1373) `module` scope fixture can be applied properly.